content/en/tracing/trace_collection/library_config/_index.md

```diff
@@ -40,6 +40,49 @@ The following configuration options behave consistently across the latest versio
 **Supported Input**: Boolean <br>
 **Description**: Enables or disables sending traces from the application.
 
+`DD_TRACE_SAMPLE_RATE`
+: **Default**: `-1`. If unset, the tracer defers to the Datadog Agent to control sample rate. <br>
+**Supported Input**: A float between 0.0 and 1.0, inclusive.  <br>
+**Caveats**: This variable is deprecated in favor of `DD_TRACE_SAMPLING_RULES`, which provides more flexible and granular sampling control.  <br>
+**Description**: Controls the trace ingestion sample rate between the Datadog Agent and the backend. Must be a float between 0.0 and 1.0, where 1.0 means all traces are sent to the backend and 0.0 means none are sent. This setting applies globally to all traces and does not support per-service or per-operation targeting.
```

Comment:
```suggestion
**Description**: Controls the trace ingestion sample rate between the Datadog Agent and the backend. Must be a number between 0.0 and 1.0, where 1.0 means all traces are sent to the backend and 0.0 means none are sent. This is precise up to 6 digits, applies globally to all traces, does not support per-service or per-operation targeting. 
```

I also added a bit about the precision here since anything past 6 digits can run into floating point rounding issues where the sampling rate may be slightly over/under, but not sure if the customer would care